nemo/core/classes/common.py

```diff
@@ -530,6 +530,7 @@ def restore_from(
         return_config: bool = False,
         trainer: Optional['Trainer'] = None,
         save_restore_connector: SaveRestoreConnector = None,
```

Comment:
I thought the idea was to not have legacy args in this method. Can we move that arg to  SaveRestoreConnector constructor instead ?

Reply:
Or is it just leftover and not used ?

Reply:
Oh, I removed in modelpt and save restore connector, but missed here. Will exclude this is the refractor pr